samples/AspNetCore/GrpcServiceSample/Services/BankingService.cs

```diff
@@ -0,0 +1,168 @@
+                    var output = await GetAccount(input, context);
+                    respone.Data = TypeConverters.ToAny<Account>(output, this.jsonOptions);
+                    break;
+                case "deposit":
+                case "withdraw":
+                    var transaction = TypeConverters.FromAny<Transaction>(request.Data, this.jsonOptions);
+                    var account = request.Method == "deposit" ?
+                        await Deposit(transaction, context) :
+                        await Withdraw(transaction, context);
+                    respone.Data = TypeConverters.ToAny<Account>(account, this.jsonOptions);
+                    break;
+                default:
+                    break;
+            }
+            return respone;
+        }
+
+        /// <summary>
+        /// implement ListTopicSubscriptions to register deposit and withdraw subscriber
+        /// </summary>
+        /// <param name="request"></param>
+        /// <param name="context"></param>
+        /// <returns></returns>
+        public override Task<ListTopicSubscriptionsResponse> ListTopicSubscriptions(Empty request, ServerCallContext context)
+        {
+            var result = new ListTopicSubscriptionsResponse();
+            result.Subscriptions.Add(new TopicSubscription
+            {
+                PubsubName = "pubsub",
+                Topic = "deposit"
+            });
+            result.Subscriptions.Add(new TopicSubscription
+            {
+                PubsubName = "pubsub",
+                Topic = "withdraw"
+            });
+            return Task.FromResult(result);
+        }
+
+        /// <summary>
+        /// implement OnTopicEvent to handle deposit and withdraw event
+        /// </summary>
+        /// <param name="request"></param>
+        /// <param name="context"></param>
+        /// <returns></returns>
+        public override async Task<TopicEventResponse> OnTopicEvent(TopicEventRequest request, ServerCallContext context)
+        {
+            if (request.PubsubName == "pubsub")
+            {
+                var transaction = JsonSerializer.Deserialize<Transaction>(request.Data.ToStringUtf8(), this.jsonOptions);
+                if (request.Topic == "deposit")
+                    await Deposit(transaction, context);
+                else
+                    await Withdraw(transaction, context);
+            }
+           
+            return await Task.FromResult(default(TopicEventResponse));
+        }
+
+        /// <summary>
+        /// GetAccount
+        /// </summary>
+        /// <param name="input"></param>
+        /// <param name="context"></param>
+        /// <returns></returns>
+        public async Task<Account> GetAccount(GetAccountInput input, ServerCallContext context)
+        {
+            var state = await _daprClient.GetStateEntryAsync<Account>(StoreName, input.Id);
+
+            if (state.Value == null)
+            {
+                return null;
+            }
```

Comment:
why do we need this block? isn't this this redundant with line 128?

Reply:
fixed